pkg/importer/dataStream.go

```diff
@@ -924,9 +939,11 @@ func (d *DataStream) copy(dest string) error {
 				return err
 			}
 		} else {
-			err = util.MoveFileAcrossFs(tmpDest, dest)
-			if err != nil {
-				return err
+			if !d.isIsoImage {
```

Comment:
n this path we have already decided we need scratch space in ALL cases, so this makes no sense. Again, simply set d.qemu=true in newDataStream and isHttpQCow2 will be true for an iso image and none of this is necessary.